src/ops/init.rs

```diff
@@ -1,19 +1,20 @@
 //! Bootstrap a new lorri project
 
 use crate::ops::error::{ok_msg, ExitError, OpResult};
+use slog_scope::info;
 use std::fs::File;
 use std::io;
 use std::io::Write;
 use std::path::Path;
 
 fn create_if_missing(path: &Path, contents: &str, msg: &str) -> Result<(), io::Error> {
     if path.exists() {
-        println!("- {} {}", msg, path.display());
+        info!("skipped file"; "path" => path.to_str(), "message" => msg);
```

Comment:
```suggestion
        info!("file already exists, skipping"; "path" => path.to_str(), "message" => msg);
```

What would you think about this?